src/movieData.js

```diff
@@ -0,0 +1,326 @@
+const movieData = {
```

Comment:
Thank you for implementing our Movie data file that holds all the data for the movies we are going to display